neuralprophet/components/future_regressors/future_regressors.py

```diff
@@ -0,0 +1,27 @@
+from neuralprophet import utils
+from neuralprophet.components import BaseComponent
+
+
+class FutureRegressors(BaseComponent):
+    def __init__(self, config, id_list, quantiles, n_forecasts, device, log, config_trend_none_bool):
```

Comment:
the logger can be retrived at the beginning of a file - does not need to be passed by. (see examples of other files)